### PR TITLE
BWAA MEDIA RC3

### PR DESCRIPTION
**Pull request by dbpokorny**
(Apologies for this PR; the last one got scrambled when I tried testing out another bot and left my github credentials in the config.)

This PR is intended to support the BWAA project by adding functionality to support its development as a learning tool / entertainment center.

Example future enhancement: a http endpoint /videoctl lets the administrator add, remove, and re-order the items in the playlist, adjust the cue point (current offset into the playlist, in seconds) and pause and resume playback for all users tuning into the bot's video and audio programming. This gives the instructor the ability to review and discuss audio/video material in a flexible way. Youtube makes this possible https://developers.google.com/youtube/iframe_api_reference, but I'm not sure about soundcloud. In the future, BWAA or a specialized fork could give humans guided audio and video tours of its architecture, and allow student programmers to make changes in the sections of the codebase relevant to the current lesson. I would like to close out with remarks from a couple of "mentalists" who share insights into the relationship between education and entertainment: http://www.nytimes.com/2004/08/09/arts/a-hypersensitive-guy-playing-mind-games.html and http://creativityni.org/stories/david-meade-celebrity-mentalist

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* d3f1a1391ae061f6b11b36211850bed796326a7e

**Comment by botwillacceptanything** [comment]
:+1:

**Comment by korczis** [comment]
:-1:

**Comment by daedalus** [comment]
:-1:

**Comment by SpenserJ** [comment]
I still like the idea, but I have to :-1: until you remove `devmain` from `main.js` (should be in a separate PR) and move everything in `data/views/layouts/media.hbs` into external CSS/JS, and call it in a cleaner way. We shouldn't be duplicating the main page template.

**Comment by samis** [comment]
Remove devmain and I will vote for. Until then, :-1:

File: main.js [hunk 1]
```diff
@@ -82,6 +83,25 @@
...
+            // greet the other bots
+            talk.speak();
+
+            // Allow the voting system to bootstrap and begin monitoring PRs.
+            voting.initialize();
+        });
+    }
+
+    // Last thing before commit: change devmain back to main
+
```
Comment (shalecraig) [inline main.js]: Please remove this, then re-commit.

**Comment by shalecraig** [comment]
:-1:

**Comment by Prior99** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
#### :-1: The vote failed. This PR will now be closed.

----
**Tallies:**
:+1:: 2 (25%) 
:-1:: 6 (75%)

**Pull request closed without merging**